src/argbash.m4

```diff
@@ -35,12 +34,11 @@ cleanup()
 # $1: What (string) to pipe to autom4te
 run_autom4te()
 {
-	printf '%s\n' "$1" \
-		| autom4te "${DEBUG[@]}" -l m4sugar -I "$m4dir"
```

Comment:
I prefer to have a binary operator at the beginning of the line for the [same reason as PEP8](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator).

Reply:
Again beautify cleaning things up, I can change that behavior and will do so in the future.  For this case, might as well make it a single line, it isn't even 80 chars long.